Formula/avr-gcc@5.rb

```diff
@@ -100,8 +103,8 @@ def install
       --with-bugurl=https://github.com/osx-cross/homebrew-avr/issues
     ]
 
-    # Avoid reference to sed shim
-    args << "SED=/usr/bin/sed"
+    # Avoid reference to sed shim XXX WHY?
+    # args << "SED=/usr/bin/sed" if OS.mac?
```

Comment:
This is from the gcc formula in brew/core: https://github.com/Homebrew/homebrew-core/blob/3551ad382662c55ea76d4fd5f5d3054e2938f5af/Formula/gcc.rb#L111

I want to keep this formula as close as brew/core as possible, so I don't want to depend on gnu-sed, thus adding another dependency that could break things in the future.

The best course of action would be as simple as what you wrote:

```ruby
args << "SED=/usr/bin/sed" if OS.mac?
```

Reply:
Ah… indeed this is what [`Homebrew/linuxbrew-core`](https://github.com/Homebrew/linuxbrew-core) does, which explains why the `gcc` formula worked fine for me:

https://github.com/Homebrew/linuxbrew-core/blob/2015728ae4cd36caf830543ff408e5443ee704d0/Formula/gcc.rb#L105-L125

Took me some time to realise that _my_ `Homebrew/Library/Taps/homebrew/homebrew-core` actually points _there_, instead of `Homebrew/homebrew-core`.

I opened Homebrew/brew#10802 if you're interested in following. Once that's sorted & merged, all of this `sed` shim nonsense can just disappear.